beacon-chain/cache/active_balance.go

```diff
@@ -0,0 +1,104 @@
+// +build !libfuzzer
```

Comment:
Should we add an active_balance_disabled.go file for fuzzing?

Reply:
When it's needed. (ie using it run time). Next PR